Changes

```diff
@@ -22,6 +22,9 @@ Working version
   (Stephen Dolan and Jacques-Henri Jourdan, review by Damien Doligez,
    KC Sivaramakrishnan and Xavier Leroy)
 
+- #??: Make tuple matching optimisation apply to Lswitch and Lstringswitch.
+  (Stephen Dolan, review by ??)
```

Comment:
You need to update the remaining question marks.

Reply:
Thanks!